frb_example/pure_dart_multi/rust/src/block_specific_module/for_api_block_2.rs

```diff
@@ -0,0 +1,19 @@
+/// This is a struct only used in API block 2 for test, but not defined in block file
+pub struct StructOnlyForBlock2 {
+    pub id: i16,      // in-built type only used in API block 2 for test
+    pub num: f64,     // in-built type in all API blocks for test
+    pub name: String, // struct type used in all API blocks for test
+}
+impl StructOnlyForBlock2 {
+    #[allow(unused)]
+    /// the parameter type `u16 for `num` is only used for struct
+    /// used in a specific API block but not defined in the API block(like `StructOnlyForBlock1`,`StructOnlyForBlock3`),
+    /// for testing shared type(`u16`) within a no-shared struct method
+    pub fn test_method(&self, message: String, num: u16) -> String {
```

Comment:
off-topic (just mark as resolved after your read it): it reminds me of adding other types of tests, which may make the test easier... but anyway that's another question
maybe related: https://github.com/fzyzcjy/flutter_rust_bridge/issues/1210

Reply:
Similar as https://github.com/fzyzcjy/flutter_rust_bridge/pull/1145#discussion_r1322496303.